app/assets/javascripts/admin/addon/components/form-template/form.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="form-templates--form">
+  <div class="control-group">
+    <TextField
+      @value={{this.templateName}}
+      @name="template-name"
+      @placeholderKey="admin.form_templates.new_template_form.name.placeholder"
+    />
+  </div>
+
+  <div class="control-group form-templates--quick-insert-field-buttons">
+    <span>
+      {{I18n "admin.form_templates.quick_insert_fields.add_new_field"}}
+    </span>
+    <DButton
+      @class="btn-flat btn-icon-text"
+      @icon="check-square"
+      @label="admin.form_templates.quick_insert_fields.checkbox"
+      @title="admin.form_templates.quick_insert_fields.checkbox"
+      @action={{this.onInsertField}}
+      @actionParam="checkbox"
+    />
+    <DButton
+      @class="btn-flat btn-icon-text"
+      @icon="grip-lines"
+      @label="admin.form_templates.quick_insert_fields.short_answer"
+      @title="admin.form_templates.quick_insert_fields.short_answer"
+      @action={{this.onInsertField}}
+      @actionParam="input"
+    />
+    <DButton
+      @class="btn-flat btn-icon-text"
+      @icon="align-left"
+      @label="admin.form_templates.quick_insert_fields.long_answer"
+      @title="admin.form_templates.quick_insert_fields.long_answer"
+      @action={{this.onInsertField}}
+      @actionParam="textarea"
+    />
+    <DButton
+      @class="btn-flat btn-icon-text"
+      @icon="chevron-circle-down"
+      @label="admin.form_templates.quick_insert_fields.dropdown"
+      @title="admin.form_templates.quick_insert_fields.dropdown"
+      @action={{this.onInsertField}}
+      @actionParam="dropdown"
+    />
+    <DButton
+      @class="btn-flat btn-icon-text"
+      @icon="cloud-upload-alt"
+      @label="admin.form_templates.quick_insert_fields.upload"
+      @title="admin.form_templates.quick_insert_fields.upload"
+      @action={{this.onInsertField}}
+      @actionParam="upload"
+    />
+    <DButton
+      @class="btn-flat btn-icon-text"
+      @icon="bullseye"
+      @label="admin.form_templates.quick_insert_fields.multiple_choice"
+      @title="admin.form_templates.quick_insert_fields.multiple_choice"
+      @action={{this.onInsertField}}
+      @actionParam="multiselect"
+    />
+  </div>
+
+  <div class="control-group">
+    {{! TODO(@keegan): Switch to <AceEditor /> after fixing visibility issues }}
```

Comment:
Having some issues around getting `<AceEditor />` to show. It gets added to the DO< but isn't visible. With some tweaking the CSS in the console, I can make it appear, but since the CSS is generated in-line, need to do some more digging into how to fix this issue.

I want to use `<Ace Editor/>` here but I don't want it to be a blocker for building the core features, so I'm delaying this for later.